services/web/server/src/simcore_service_webserver/director_v2/_comp_run_collections_service.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+from aiohttp import web
+from models_library.computations import CollectionRunID
+
+from ..db.plugin import get_database_engine
+from . import _comp_runs_collections_repository
+from ._comp_runs_collections_models import CompRunCollectionDBGet
+
+_logger = logging.getLogger(__name__)
+
+
+async def create_comp_run_collection(
+    app: web.Application,
```

Comment:
MINOR: keep positional the app arguments and the rest keyword argumbent `(app, *, kw1, kw2)`